<ui version="4.0" >
 <class>URLPicPreviewPrefsUI</class>
 <widget class="QWidget" name="URLPicPreviewPrefsUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Picture Preview Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QCheckBox" name="restrictPreviews" >
        <property name="whatsThis" >
         <string>If checked, only the selected number of pictures are downloaded and displayed as previews.
Otherwise, an unlimited number of previews are generated.</string>
        </property>
        <property name="text" >
         <string>&amp;Restrict number of previews:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KIntSpinBox" name="previewAmount" >
        <property name="whatsThis" >
         <string>The maximum number of pictures to download and preview.</string>
        </property>
        <property name="value" >
         <number>2</number>
        </property>
        <property name="maxValue" stdset="0" >
         <number>4096</number>
        </property>
        <property name="minValue" stdset="0" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Scaling</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QCheckBox" name="enableScaling" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>If checked, preview images get scaled down to the width specified.
Otherwise, the images are not scaled at all and appear in their original size.</string>
        </property>
        <property name="text" >
         <string>Scale &amp;to:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KIntSpinBox" name="previewScaleWidth" >
        <property name="whatsThis" >
         <string>The width to which images should be scaled down. Smaller images will appear in their original size.</string>
        </property>
        <property name="suffix" >
         <string> pixel</string>
        </property>
        <property name="value" >
         <number>99</number>
        </property>
        <property name="maxValue" stdset="0" >
         <number>1024</number>
        </property>
        <property name="minValue" stdset="0" >
         <number>16</number>
        </property>
        <property name="lineStep" stdset="0" >
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="linkPreviewGroup" >
     <property name="title" >
      <string>Link Preview</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="enableLinkPreview" >
          <property name="whatsThis" >
           <string>Creates a preview picture of the link.</string>
          </property>
          <property name="text" >
           <string>Enable lin&amp;k preview</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Abort creating previe&amp;w after</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>timeoutLinkPreview</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntNumInput" name="timeoutLinkPreview" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>After the specified number of seconds the creation of the preview is aborted automatically.
Use 0 seconds to never abort the creation.</string>
          </property>
          <property name="value" >
           <number>10</number>
          </property>
          <property name="suffix" >
           <string> seconds</string>
          </property>
          <property name="minValue" stdset="0" >
           <number>0</number>
          </property>
          <property name="maxValue" stdset="0" >
           <number>3600</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>restrictPreviews</tabstop>
  <tabstop>previewAmount</tabstop>
  <tabstop>enableScaling</tabstop>
  <tabstop>previewScaleWidth</tabstop>
  <tabstop>enableLinkPreview</tabstop>
  <tabstop>timeoutLinkPreview</tabstop>
 </tabstops>
 <includes>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>enableScaling</sender>
   <signal>toggled(bool)</signal>
   <receiver>previewScaleWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrictPreviews</sender>
   <signal>toggled(bool)</signal>
   <receiver>previewAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableLinkPreview</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeoutLinkPreview</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
